docker-compose.env.example

```diff
@@ -32,9 +32,9 @@ SECRET_KEY_BASE=changemechangemechangeme"
 # you should comment out or remove the mailhog service from docker-compose.yml,
 # too
 OSEM_EMAIL_ADDRESS="osem@mailhog"
-OSEM_SMTP_AUTHENTICATION="login"
-OSEM_SMTP_ADDRESS="mailhog"
-OSEM_SMTP_PORT="1025"
```

Comment:
I assume this is the real problem, as this is parsed as a `String`, whereas without the quotes, it's an `Integer`.

Reply:
@bear454  if I understand correctly, we need to only remove the quotes for `OSEM_SMTP_PORT`, right?

@vmisson can you leave the quotes in the rest so we can get this merge? 😉